### PR TITLE
[Home Header] Improvements "Favourites"

### PR DESCRIPTION
**Pull request by Nanomani**
screenshot00014

- need "script.favourites" (Repo Kodi)

screenshot00015

- a review is needed :)

### CODE REVIEW

File: 1080i/Home.xml [hunk 1]
```diff
@@ -11,6 +11,7 @@
     <onunload condition="$EXP[HasHomemenuAutoSlideOutAnimation]">CancelAlarm(slide_out,true)</onunload>
     <onunload>ClearProperty(TMDbHelper.WidgetContainer,home)</onunload>
     <onload condition="!$EXP[HomeIsModernMultiWidgets] + !$EXP[HomeIsVerticalMultiWidgets]">SetProperty(TMDbHelper.WidgetContainer,301,home)</onload>
+    <onload condition="System.HasAddon(script.favourites)">RunScript(script.favourites)</onload>
```
Comment (Nanomani) [inline 1080i/Home.xml:14]: I m not sure if it is the better place ...

**Comment by beatmasterRS** [comment]
- no need to add a dependency, that is not required for the main skin
- check if the addon is enabled to prevent an empty string

:)

**Comment by Nanomani** [comment]
@beatmasterRS, I know that seems empty. :/

There is the same with weather :

**Comment by Nanomani** [comment]
@beatmasterRS this one is finished too
I don't know doing better :/

**Review by beatmasterRS — CHANGES_REQUESTED** [review]
Something must be wrong with this condition:

>>String.IsEqual(Container(300).ListItem.Property(defaultID),$LOCALIZE[1036])

if i set change this to 

>>true

all is working fine

**Comment by Nanomani** [comment]
> Something must be wrong with this condition:
> 
> > > String.IsEqual(Container(300).ListItem.Property(defaultID),$LOCALIZE[1036])
> 
> if i set change this to
> 
> > > true
> 
> all is working fine

Something is wrong ?  
Tested with english french and german language. All works well ...
But if you think an other way is better please you can change the code. :)

I have use this code because for favourite the default ID is 1036 like this :  
`<property name="defaultID">1036</property>`

**Comment by beatmasterRS** [comment]
Must the homemenu entry also set to favourites, or has only the widget content set to favourites?

**Comment by Nanomani** [comment]
Only the main menu must set to "Favourites". The widgets can be what you want.

screenshot00010

screenshot00002

screenshot00003

screenshot00010

**Comment by Nanomani** [comment]
Damned you are right. I delete the widget then this don't works. :/ 
Okay I look this ...

**Comment by beatmasterRS** [comment]
ah okay, then this was my fault, sorry, just saw on your screenshot the widget content :D

**Comment by Nanomani** [comment]
I thinks is good this time :)

**Pull request merged**